lib/uefi/uefi.cpp

```diff
@@ -139,11 +140,52 @@ int load_sections_and_execute(bdev_t *dev,
     return -static_cast<int>(status);
   }
 
+  struct EFI_LOADED_IMAGE_PROTOCOL *efiLoadedImageProtocol = NULL;
```

Comment:
can we move all of this to debug_support.cpp ? We create a `setup_debug_support` function, and pass necessary information(such as system table address) to this function